make.sh

```diff
@@ -136,6 +157,6 @@ case "$1" in
   "linux32" ) build_linux32;;
   "msvc_update_genfiles" ) msvc_update_genfiles;;
```

Comment:
This one I have no idea about its description. Could you give a suggestion?

Reply:
this is to support MSVC build, which cannot automate this copying step itself.